modules/openapi-generator/src/main/resources/php/composer.mustache

```diff
@@ -26,11 +26,18 @@
         "ext-curl": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "guzzlehttp/guzzle": "^6.2"
+        "php-http/discovery": "^1.14",
+        "psr/http-factory": "^1.0",
```

Comment:
`http-factory-implementation`, no?

Reply:
Just realized, we don't require `psr/http-client-implementation`?

Reply:
I will add these, and also found a few direct dependencies that were not directly required but came as transient dependencies